impeller/docs/renderdoc_frame_capture.md

```diff
@@ -17,15 +20,19 @@
 
    Building a "debug_unopt" build ensures that you have tracing enabled. Without this, RenderDoc will not have much to show.
 
-3. Start RenderDoc, and click the "Launch Application" button.
+3. Start RenderDoc using the `qrenderdoc` command, and (if necessary) select "Launch Application" button from the menu:
```

Comment:
On Windows this is a normal GUI application that isn't launched from the command line. Should we split into two steps. Something like:

```suggestion
3. Start RenderDoc. On Linux, use the `qrenderdoc` command
4. If necessary, select "Launch Application" button ....
```

Reply:
I tweaked it a bit for ya. :-)